common/src/plane.rs

```diff
@@ -81,7 +80,7 @@ impl Plane<f64> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::math::translate_along;
+    use crate::math::{origin, translate_along};
```

Comment:
This change seems unrelated to the commit message.

Reply:
Surprisingly, this change is related. Since `origin` is no longer used above in `Plane<f64>::from`, `rustc` throws a warning about the import, but a test still uses it, so the import has to be moved down here.